gem/settings/base.py

```diff
@@ -427,7 +427,7 @@
 
 
 AUTHENTICATION_BACKENDS = [
-    'gem.backends.GemModelBackend',
+    'molo.profiles.backends.MoloProfilesModelBackend',
```

Comment:
This change needs to happen in `gem/settings/production.py` as well

Reply:
Wait actually why do we need to set this in `production.py` at all? It's exactly the same as here.